Rubberduck.Resources/Inspections/InspectionResults.resx

```diff
@@ -435,8 +435,8 @@ In memoriam, 1972-2018</value>
     <comment>{0} Method kind, {1} Method name</comment>
   </data>
   <data name="ImplementedInterfaceMemberInspection" xml:space="preserve">
-    <value>Interface class module '{2}' contains a concrete implementation for {0} '{1}'.</value>
-    <comment>{0} Method kind, {1} Method name, {2} interface class module name</comment>
+    <value>Interface '{0}' contains a concrete implementation for the following member(s): {1}</value>
+    <comment>{0} interface class name; {1} formatted string of results</comment>
```

Comment:
That comment should be more specific: something like _list of implemented members_

Reply:
Please remove all translations of this resource, unless you can change those as well. Otherwise, the people translating the resources will not realize that something has to be done.